src/axom/primal/operators/squared_distance.hpp

```diff
@@ -131,7 +131,7 @@ AXOM_HOST_DEVICE inline double squared_distance(const BoundingBox<T, NDIMS>& A,
     return v.squared_norm();
   }
 
-  return std::numeric_limits<T>::quiet_NaN();
+  return std::numeric_limits<T>::max();
```

Comment:
Value judgement -- for invalid bounding boxes, we now return `std::numeric_limits<T>::max()` instead of NaN in `squared_distance(bbox,bbox)`.
This seemed less likely to get in users way, since we don't need to always check for `isNaN()`.

Thoughts?

Reply:
Sounds good.  I'd like to go with whatever works best for the majority of users.